index.html

```diff
@@ -334,6 +351,83 @@ <h6 id="text">
           <div class="shape"></div>
         </div>
       </footer>
+      <section class="modal-section" id="main-project">
+        <div class="modal">
+          <h3>Multi Post Stories</h3>
+          <img src="image/x-icon.png" class="close-btn" alt="close button" />
+          <ul id="tech-used">
+            <li class="item">html</li>
+            <li class="item">Bootstrap</li>
+            <li class="item">Ruby on rails</li>
+          </ul>
+          <div class="modal-content">
+            <img
+              src="image/snapshot.jpg"
+              class="multi-image"
+              alt="Multi post stories image"
+            />
+            <div class="modal-details">
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it to make a type specimen book. It has
+                survived not only five centuries, but also the leap into
+                electronic typesetting, remaining essent
+              </p>
+              <ul class="modal-btn">
+                <li>
+                  <button class="project-button" type="button">See Live</button>
+                </li>
+                <li>
+                  <button class="project-button" type="button">
+                    See source
+                  </button>
```

Comment:
- [ ] Please add icons on buttons as provided in Figma:

Figma  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/151350712-78ce5fd2-98b6-4803-bfd3-dbbbe6159712.png) | ![image](https://user-images.githubusercontent.com/77038610/151350653-f7689bba-fa45-4fff-b8a5-a85e8ceac7cb.png)